book/proposal/proposal.md

```diff
@@ -0,0 +1,99 @@
+
+- Providing a better understanding of the entire domain.
+- A clearer path to a domain language.
+- Understanding problem nuance.
+- Gaining lessons learned of the user.
+- Clear prioritization of tasks.
+- Identifying focused solutions.
+- Providing insight into how an implementation may evolve.
+
+Following the path of a team replacing a legacy application with a new application, this book will explore tools that can be utilized to break down a domain and better communicate with domain experts. The book will introduce a number of design thinking exercises and explain how they can be applied in different scenarios to help agile developers craft clearer implementations.
+
+## What this book will cover:
+
+- Introduce the concept of design thinking.
+- Introduce the following design thinking methodologies from the perspective of the developer:
+    - Stakeholder mapping.
+    - Interviewing.
+    - Job Shadowing.
+    - Rose Bud Thorn.
+    - Affinity Mapping.
+    - Experience diagramming.
+    - Importance/Difficulty matrix.
+    - Feature Voting.
+    - Creative matrixes.
+    - Technical prototyping.
+    - Storyboarding UI's
+    - Stakeholder validation.
+    - More??
+- Introduce domain driven design concepts:
+    - Ubiquitous language.
+    - Context bounding.
+    - Domain events.
+- Outline effective tools for using design thinking to create ubiquitous language, bounded contexts and domain events.
+- Outline rules for effective design thinking on agile teams.
+- Discuss ideas for bringing these methods into teams that are not familiar with design thinking.
+
+## Audience:
+This book is suitable for developers, managers, product manager, project managers, designers and anyone looking for methods to bring implementation closer to a domain.
+
+## Why is your book different?
+
+This book will offer design thinking tools set aimed at software development, with a minor focus on domain driven design. It will introduce software developers to all the phases of design thinking and teach them skills to help solve problems in more creative ways, skills emphasized include:
+
+- Communicating effectively with stakeholders.
+- Identifying when to utilize design thinking methods and when not to.
+- The discovery, ideate, prototype phases of design thinking.
+- Individual design thinking methodologies and when to use them in the software development lifecycle. 
+- How to run better retros.
+- How to evaluate what to build given competing priorities.
+- Understanding the business domain they work in the day-to-day.
+- Building long-lived communication tools that impact internal/external communication and code design.
+
+
+## Why we should get excited about your topic:
+
+In the Cathedral and the Bazaar a number of important ideas were set forth related to working more closely with our stakeholder:
+
+1. Treating your users as co-developers is your least-hassle route to rapid code improvement and effective debugging.
```

Comment:
Although Markdown will take care of numbering during render, they'll probably read this as plain text. I'd suggest either number using incrementing numbers or (preferably) just use an unordered list.